app/common/directives/filter-system/filter-role.html

```diff
@@ -1,5 +1,5 @@
 <fieldset>
-    <legend translate>app.roles</legend>
+    <label translate>app.roles</label>
```

Comment:
Possibly invalid, but the rendering actually works

Reply:
:/ uh I seem to remember that using label broke the styles since they were designed for legend. Why was this needed?

Reply:
Legend styles are broken. Only labels are styled properly in the dropdowns

Reply:
You can see the broken styles here: http://preview.ushahidi.com/platform-pattern-library/develop/assets/html/5_layouts/settings-users.html 
Or just on develop

Reply:
ah. ok 👍